### PR TITLE
Add basic app + styling for Connected Accounts.

### PR DESCRIPTION
**Pull request by juliasolorzano**
## Description

This pull request is for adding a way for an End User(Veteran) to manage the authorized/connected accounts to their VA.gov profile. It is part of the work that the VA API Platform team is doing to add Veteran Verification OAuth as a service to our End Users.

Fixes https://github.com/department-of-veterans-affairs/vets-contrib/issues/178

## Testing done
- [x] Accessibility testing
- [x] Cross browser testing
- [x] Write tests 😄 

## Screenshots
See https://github.com/department-of-veterans-affairs/vets-contrib/issues/178#issuecomment-432248738

## Acceptance criteria
- [x] End User is able to view a list of their authorized/connected accounts
- [x] End User is able to revoke access to their authorized connected accounts

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by juliaelman** [comment]
A few notes about this pull request (didn't want to put in the description):

- @edpaget and I are tag teaming on this work. ^5 He'll be handling (or delegating) the rest of the work to remove the [WIP].
- I left some of the personalization/profile modules in various places. Seemed like we might use these in this work, but wasn't 100% clear. Might be useful.
- There is a PR dependency over on the new/shiny `vagov-content` repo to complete this work, see https://github.com/department-of-veterans-affairs/vagov-content/pull/43. It seemed like we were required to have this file in order to get this to work properly in the current build system. Might be nice we could not have this as a dependency, as 95% of the content is dynamic. However, this is my first PR on this project :tada: and am learning how all the pieces fit together.
- The current route is `/connected-accounts`, but should be `/account/connected-accounts`.

File: src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                 <img
+                   src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/USAJOBS.gov_logo.svg/2000px-USAJOBS.gov_logo.svg.png"
+                   alt="VA logo"
+                   width="100"
+                   />
+                </a>
+               </th>
+               <th>
+                 <a href="https://usajobs.gov">USAJOBS Profile</a> <br />
+                 Last used within the past 2 days
```
[the remark this answers is not in the document]
Comment (juliaelman) [reply src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx]: 🤔 did you mean that it "doesn't" seem like Okta gives us this information. If so, then yes. We could change the copy to read: `Connected on <M d, Y>`

[the remark this answers is not in the document]
Comment (edpaget) [reply src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx]: Yes sorry for the typo. Will change to that

**Review by edpaget — APPROVED** [review]
:ship:

**Comment by juliaelman** [comment]
Talked to @edpaget. Holding off from merging in until we sort out the url pathing issue. Right now it goes to `connected-accounts`. It should go to `account/connected-accounts`.

**Pull request merged**